After 'Sign-In' Page 'Madhav.M'/main.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
Please delete the "After 'Sign-In' Page 'Madhav.M'" folder. All the code must be within the `src` folder and assets (e.g., images) in the `public` folder. No new folders should be created at the top level.